README.md

```diff
@@ -564,6 +564,60 @@ public class AWSSecretsManagerPluginSample {
+
+### Loading the Read-Write Splitting Plugin
+
+The read-write splitting plugin is not loaded by default. To load the plugin, set the `connectionPluginFactories` connection parameter:
+
+```
+final Properties properties = new Properties();
+properties.setProperty("connectionPluginFactories", ReadWriteSplittingPluginFactory.class.getName());
+```
+
+If you would like to load the read-write splitting plugin alongside the failover and enhanced failure monitoring plugins, the read-write splitting plugin must be the first plugin in the connection chain, otherwise failover exceptions will not be properly processed by the plugin:
+
+```
+final Properties properties = new Properties();
+properties.setProperty(
+        "connectionPluginFactories",
+        String.format("%s,%s,%s",
+                ReadWriteSplittingPluginFactory.class.getName(),
+                FailoverConnectionPluginFactory.class.getName(),
+                NodeMonitoringConnectionPluginFactory.class.getName())
+```
+
+### Reader load-balancing
+
+The plugin can also load-balance queries among available reader instances by enabling the `loadBalanceReadOnlyTraffic` connection parameter. This parameter is disabled by default. To enable it, set the following connection parameter:
+
+```
+final Properties properties = new Properties();
+properties.setProperty("loadBalanceReadOnlyTraffic", "true");
+```
+
+Once this parameter is enabled, load-balancing will automatically be performed for reader instances when the connection has been set to read-only mode via `JdbcConnection#setReadOnly`. Load-balancing is performed by randomly selecting new reader instances at transaction boundaries.
+
+### Using the Read-Write Splitting Plugin against RDS/Aurora clusters
+
+When using the read-write splitting plugin against RDS or Aurora clusters, the plugin automatically acquires the cluster topology by querying the cluster. Because of this functionality, you do not have to supply multiple instance URLs in the connection string. Instead, users should only supply the URL for the initial instance they wish to connect to.
+
+### Using the Read-Write Splitting Plugin against non-RDS clusters
+
+If you would like to use the read-write splitting plugin against a cluster that is not RDS or Aurora, the plugin will not be able to automatically acquire the cluster topology. Instead, you must supply the topology information to the plugin by specifying multiple instance URLs in the connection string. Instance URLs should be supplied via a comma-separated list. The first instance in the list must be the writer instance:
```

Comment:
Should the example that follows include a ':aws:' component? How about:

If you are using the read-write splitting plugin against a cluster that is not hosted on RDS or Aurora, the plugin will not be able to automatically acquire the cluster topology. Instead, you must supply the topology information in the connection string as a comma-delimited list of multiple instance URLs. The first instance in the list must be the writer instance:

Reply:
Yes it should. Even if the database is not RDS, the ':aws:' component of the protocol will enable the features that AWS has added instead of using the regular community driver features (read-write splitting, failover etc - although failover is not supported for non-RDS instances).

Good suggestion, I'll make that change!

Reply:
TY!